packages/ui/components/StickyHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import classnames from 'classnames'
+import { PropsWithChildren } from 'react'
+
+export interface StickyHeaderProps
+  extends React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLElement>,
+      HTMLElement
+    >,
+    PropsWithChildren {
+  variant?: 'primary' | 'light' | 'dark'
+  size?: 'sm' | 'md' | 'lg'
+  shadow?: 'sm' | 'md' | 'lg'
```

Comment:
doesn't need to be dynamic. alternatively, we can just use a shadow for the entire design system and stick with it (same with what we're doing for border radius)